common/src/main/java/com/microsoft/identity/common/internal/platform/Device.java

```diff
@@ -22,124 +22,35 @@
 //  THE SOFTWARE.
 package com.microsoft.identity.common.internal.platform;
 
-import android.os.Build;
-import android.text.TextUtils;
-
-import com.microsoft.identity.common.adal.internal.AuthenticationConstants;
 import com.microsoft.identity.common.exception.ClientException;
-import com.microsoft.identity.common.logging.DiagnosticContext;
 
 import java.io.IOException;
 import java.security.KeyStoreException;
 import java.security.NoSuchAlgorithmException;
 import java.security.cert.CertificateException;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 
 /**
- * Helper class to add additional platform specific query parameters or headers for the request sent to sts.
+ * A static class holding device data.
```

Comment:
I don't really see the `static` keyword on the class though

Reply:
It's inferred for top-level classes, right?  But in this case, I think the missing word is utility, meaning all the methods are static and the class is never expected to be instantiated?

Reply:
@AdamBJohnsonx correct. I don't think the word 'utility' fits in this context though, since this class is providing information about the device. It's not really doing any work.

Reply:
And access to device-specific features on line 149.  Nevertheless, a static utility class is /normally/ a class where all the methods are static.  This one would seem to qualify.

Reply:
I actually would like to move that function out of this class, but decided not to do it to scope the change down.

I don't think it belongs here.